### PR TITLE
add README.md

### PR DESCRIPTION
**Pull request by abhinavdahiya**
/cc @crawford

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+## Building and Publishing CVO
+
+```sh
+./hack/build-image.sh && REPO=<your personal repo (quay.io/ahbinavdahiya | docker.io/abhinavdahiya)> ./hack/push-image.go
+```
+
+This builds image using `podman` locally using following command:
+
+```sh
+podman build -t "cluster-version-operator:${VERSION}" -f Dockerfile
```
Comment (wking) [inline README.md]: Do we care?  Folks who do can look in the script.  Folks who don't can ignore these details.  I'd rather drop this shell snippet and use:

> This builds image locally and then pushes `${VERSION}` and `latest` tags to `${REPO}/origin-cluster-version-operator`.

Along with some talk about `${VERSION}`; maybe:

> `${VERSION}` encodes the Git commit used to build the images.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+1. Run the following command to get render the release-payload contents to `/tmp/cvo`
+
+```sh
+podman run --rm -ti \
+    -v /tmp/cvo:/tmp/cvo:z \
+    <release image> \
+        render \
+        --output-dir=/tmp/cvo \
+        --release-image="<release image>"
```
Comment (wking) [inline README.md]: Do we have real location we can plug in here?

File: hack/push-image.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+REPO=${REPO:-"openshift"}
+
+if [ -z ${VERSION+a} ]; then
+        print_info "Using version from git..."
+        VERSION=$(git describe --abbrev=8 --dirty --always)
+fi
+
+set -x
+podman push "cluster-version-operator:${VERSION}" "${REPO}/origin-cluster-version-operator:${VERSION}"
+podman push "cluster-version-operator:${VERSION}" "${REPO}/origin-cluster-version-operator:latest"
```
Comment (wking) [inline hack/push-image.sh]: nit: you're missing a trailing newline here.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+# Cluster Version Operator (CVO)
+
+## Building and Publishing CVO
+
+```sh
+./hack/build-image.sh && REPO=<your personal repo (quay.io/ahbinavdahiya | docker.io/abhinavdahiya)> ./hack/push-image.go
```
[the remark this answers is not in the document]
Comment (abhinavdahiya) [reply README.md]: i think current is fine for now.

File: README.md (hunk 2 above)
Comment (abhinavdahiya) [reply README.md]: added another line below with options.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+        --release-image="<release image>"
+```
+
+`<release image>` can be personal release image generated using [this](building-release-image-using-local-cvo) or Origin's release image like `registry.svc.ci.openshift.org/openshift/origin-release:v4.0`.
+
+## Installing CVO and operators in cluster.
+
+1. Use CVO `render` to render all the manifests from release-payload to a directory. [here](#using-cvo-to-render-the-release-payload-locally)
+
+2. For example if you rendered all the manifests in `/tmp/cvo` directory. Run `oc create -f /tmp/cvo`.
```
Comment (yifan-gu) [inline README.md]: Maybe something like:
```
2. Create the operators with the manifests, for example ...
```

**Comment by yifan-gu** [comment]
@abhinavdahiya 2 nits

File: README.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+## Building and Publishing CVO
+
+```sh
+./hack/build-image.sh && REPO=<your personal repo (quay.io/ahbinavdahiya | docker.io/abhinavdahiya)> ./hack/push-image.go
+```
+
+1. This builds image locally and then pushes `${VERSION}` and `latest` tags to `${REPO}/origin-cluster-version-operator`.
+
+2. ${VERSION}` encodes the Git commit used to build the images.
```
[the remark this answers is not in the document]
Comment (abhinavdahiya) [reply README.md]: done.

File: README.md (hunk 5 above)
Comment (abhinavdahiya) [reply README.md]: done.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+```sh
+podman run --rm -ti \
+    -v /tmp/cvo:/tmp/cvo:z \
+    <release image> \
+        render \
+        --output-dir=/tmp/cvo \
+        --release-image="<release image>"
+```
+
+`<release image>` can be personal release image generated using [this](building-release-image-using-local-cvo) or Origin's release image like `registry.svc.ci.openshift.org/openshift/origin-release:v4.0`.
```
Comment (wking) [inline README.md]: nit: `](` -> `](#`

**Comment by wking** [comment]
Maybe a line or two at the top to set the context?

> The CVO manages individual nodes in an OpenShift cluster, performing rolling upgrades and... (whatever else we do) based on admin changes to (the places we look for upstream config).

Or something :p

**Comment by wking** [comment]
/lgtm
\We can always add more later ;).

**Pull request merged**